src/components/shop-app.js

```diff
@@ -217,9 +203,13 @@ class ShopApp extends connect(store)(LitElement) {
     <app-header role="navigation" id="header" effects="waterfall" condenses="" reveals="">
       <app-toolbar>
         <div class="left-bar-item">
-          <paper-icon-button class="menu-btn" icon="menu" on-click="${_ => this.drawerOpened = true}" aria-label="Categories">
+          <paper-icon-button class="menu-btn" icon="menu"
+              on-click="${_ => this.drawerOpened = true}"
+              aria-label="Categories"
+              hidden="${page === 'detail'}">
```

Comment:
What about also checking for `_smallScreen`, e.g. `hidden="${!_smallScreen || page === 'detail'}"`? After that we can remove these: [style1](https://github.com/Polymer/shop/blob/86cb0423feff951ff3567c0706d6b1c71cc53531/src/components/shop-app.js#L190-L196), [style2](https://github.com/Polymer/shop/blob/86cb0423feff951ff3567c0706d6b1c71cc53531/src/components/shop-app.js#L87-L89)